actors/rozetka-count/src/tools.js

```diff
@@ -0,0 +1,45 @@
+import Apify from 'apify';
+import { NAMED_KV_STORE, STATS_KEY } from './consts.js';
+
+const { utils: { log } } = Apify;
+
+export const purgeStatsStore = async () => {
```

Comment:
Why we use named stores here? Do we need to store something for in-between runs?

Reply:
I use a named store to preserve the results of the counting in case of migration of the docker container. So that, when the migrated container restarts, it takes the initial count value from the store. That's why I clear this store when the crawling is completed, so that another separate crawling cycle doesn't mix up with the previous actor's run resulting count value. Is there a better solution for this or is there any dangers with this approach?

Reply:
See my other reply :)